src/styles/blocks/nav.scss

```
@@ -0,0 +1,41 @@
+.nav {
+  &__list {
+    margin: 0;
+    display: flex;
+    list-style-type: none;
+    padding: 0;
+  }
+  
+  &__link {
+
+    font-weight: 500;
+    display: block;
+    text-decoration: none;
+    font-size: 12px;
+    color: $c-main-dark;
+    padding-top: 23px;
+
+    &--active {
+      color: $c-blue-main;
+  
+      &::after {
+        padding-left: 0;
+        margin-top: 18.5px;
+        display: block;
+        content: "";
+        width: 36px;
```

Use 100% width of the active link

```suggestion
        width: 100%;
```